src/pages/ThorChainLP/AvailablePools.tsx

```diff
@@ -121,60 +112,32 @@ export const AvailablePools = () => {
       },
       {
         Header: translate('pools.volume24h'),
-        // TODO(gomes): expose me in the Pool interface so this can be used for sorting
-        // accessor: 'volume24h',
+        accessor: 'volume24hFiat',
         display: { base: 'none', lg: 'table-cell' },
-        Cell: ({ row }: { value: string | undefined; row: RowProps }) => {
-          const pool = row.original
-
-          const { data: swapsData } = useQuery({
-            ...reactQueries.midgard.swapsData(pool.asset, 'hour', 7 * 24),
-            select: selectSwapsData,
-          })
-
-          const volumeStats = useMemo(() => {
-            if (!swapsData) return
-            return getVolumeStats(swapsData, runeMarketData.price)
-          }, [swapsData])
-
-          return (
-            <Skeleton isLoaded={!!volumeStats}>
-              <Skeleton isLoaded={!!volumeStats} display={mobileDisplay}>
-                <Amount.Fiat value={volumeStats?.volume24hFiat ?? '0'} />
-              </Skeleton>
+        Cell: ({ value: volume24hFiat }: { value: string | undefined; row: RowProps }) => (
+          <Skeleton isLoaded={!!volume24hFiat}>
+            <Skeleton isLoaded={!!volume24hFiat} display={mobileDisplay}>
+              <Amount.Fiat value={volume24hFiat ?? '0'} />
             </Skeleton>
-          )
-        },
+          </Skeleton>
+        ),
       },
       {
         Header: translate('pools.volume7d'),
-        // TODO(gomes): expose me in the Pool interface so this can be used for sorting
-        // accessor: 'volume7d',
+        accessor: 'volume7dFiat',
         display: { base: 'none', lg: 'table-cell' },
-        Cell: ({ row }: { value: string | undefined; row: RowProps }) => {
-          const pool = row.original
-
-          const { data: swapsData } = useQuery({
-            ...reactQueries.midgard.swapsData(pool.asset, 'hour', 7 * 24),
-            select: selectSwapsData,
-          })
-
-          const volumeStats = useMemo(() => {
-            if (!swapsData) return
-            return getVolumeStats(swapsData, runeMarketData.price)
-          }, [swapsData])
-
+        Cell: ({ value: volume7dFiat }: { value: string | undefined; row: RowProps }) => {
           return (
-            <Skeleton isLoaded={!!volumeStats}>
-              <Skeleton isLoaded={!!volumeStats} display={mobileDisplay}>
-                <Amount.Fiat value={volumeStats?.volume7dFiat ?? '0'} />
+            <Skeleton isLoaded={!!volume7dFiat}>
+              <Skeleton isLoaded={!!volume7dFiat} display={mobileDisplay}>
```

Comment:
tiniest ever nitpick, but we've been sticking with `Boolean(..)` rather than `!!..`